include/BoardData.hxx

```diff
@@ -0,0 +1,52 @@
+/*
+ *  BoardData.hxx
+ *  Copyright 2020 ItJustWorksTM
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+#ifndef SMARTCAR_EMUL_BOARDDATA_HXX
+#define SMARTCAR_EMUL_BOARDDATA_HXX
+#include <atomic>
+#include <range\v3\range\conversion.hpp>
+
+template <std::size_t N>
+
+struct FixBufferBus {
```

Comment:
Why is there an empty line between the template declaration and the class definition?